jbpm-workitems/jbpm-workitems-webservice/src/main/java/org/jbpm/process/workitem/webservice/WebServiceWorkItemHandler.java

```diff
@@ -110,12 +174,23 @@ public void executeWorkItem(WorkItem workItem,
             if (client == null) {
                 throw new IllegalStateException("Unable to create client for web service " + interfaceRef + " - " + operationRef);
             }
+
             //Override endpoint address if configured.
             if (endpointAddress != null && !"".equals(endpointAddress)) {
                 client.getRequestContext().put(Message.ENDPOINT_ADDRESS,
                                                endpointAddress);
             }
 
+            if(username != null) {
+                HTTPConduit httpConduit = (HTTPConduit) client.getConduit();
+                AuthorizationPolicy authorizationPolicy = new AuthorizationPolicy();
+                authorizationPolicy.setUserName(username);
+                authorizationPolicy.setPassword(password);
+
+                authorizationPolicy.setAuthorizationType("Basic");
+                httpConduit.setAuthorization(authorizationPolicy);
+            }
```

Comment:
maybe move this to protected method so in case someone would like to override the auth mechanism would just extend and override single method